index.html

```diff
@@ -12,30 +12,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>PORFOLIO</title>
     <link rel="stylesheet" href="styles.css">
-    <link rel = "icon" href =  "/Porfolio/pictures/bxs-radiation.svg" type = "image/x-icon">
+    <link rel = "icon" href =  "pictures/bxs-radiation.svg" type = "image/x-icon">
    
 </head>
 <body>
   <section class="prinsipal">
-    <nav class="navbar">
-      <div class="hora color" >9:45</div>
-      <img class="signal color icons" src="/Porfolio/pictures/Wifi.svg" alt="">
-      <img class="wifi color icons" src="/Porfolio/pictures/Cellular Connection.svg" alt="">
-      <img class="battery color icons" src="/Porfolio/pictures/Battery.svg" alt="">
-      <div class="name color" >John Doe</div>
-      <img class="bxicon" src="/Porfolio/pictures/bx-menu.svg" alt="">
-    </nav>
+    <header class="navbar">
+      <p class="name color" > <a class="nameanchor" href="http://">John Doe</a> </p>
+      <img class="bxicon" src="pictures/menu.svg" alt="">
+    </header>
```

Comment:
- [ ] Hey @Eldonratzlaff, did well on implementing some of the changes requested by the previous reviewer. However, there are still some issues that I would recommend fixing.
- [ ] First your logo and the hamburger menu are not properly aligned. 

See 👇️
![Screenshot from 2022-11-04 01-03-50](https://user-images.githubusercontent.com/89228743/199863120-180ec22e-9074-4a70-8c13-d8a9282f01be.png)

To fix the issue add this to your `.navbar` class in the CSS.
```
.navbar {
    display: flex;
    justify-content: space-between;
    align-items: center;
}
```

- [ ] For the `name` class remove the ` margin: 60px 210px 0 20px;` and give it a `padding-left: 25px;`.
- [ ] For the `bxicon` class remove the `padding: 0 0 4px 10px; ` and give it a `padding-right: 25px;`